.github/workflows/tag-publish.yaml

```diff
@@ -83,8 +85,18 @@ jobs:
           username: 'oauth2accesstoken'
           password: '${{ steps.auth.outputs.access_token }}'
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
       - name: Build image
-        run: "docker build -t ${{ steps.image-name.outputs.tagged }} ."
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: false
+          load: true
+          tags: |
+            ${{ steps.image-name.outputs.tagged }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
Not sure how all of this works, but it looks like the same intent and I trust that you know that this is equivalent.

Reply:
Yup, been using it for Beehive, and this build process only affects BEEs anyway--all of Terra UI's deployments are entirely exempt from what I'm doing here